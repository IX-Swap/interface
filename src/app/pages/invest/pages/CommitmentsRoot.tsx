import { useCommitmentRouter } from 'app/pages/invest/routers/commitmentsRouter'

export const CommitmentsRoot = () => {
  const { renderRoutes } = useCommitmentRouter()
  return renderRoutes()
}
