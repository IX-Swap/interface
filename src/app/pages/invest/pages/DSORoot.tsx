import { useDSORouter } from 'app/pages/invest/routers/dsoRouter'

export const DSORoot = () => {
  const { renderRoutes } = useDSORouter()

  return renderRoutes()
}
