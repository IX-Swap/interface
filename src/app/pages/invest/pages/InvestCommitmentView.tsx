import React from 'react'
import { useCommitmentById } from 'app/pages/invest/hooks/useCommitmentById'
import { CommitmentPreview } from 'app/components/CommitmentPreview/CommitmentPreview'
import { RejectionMessage } from 'app/pages/authorizer/components/RejectionMessage'
import { useParams } from 'react-router-dom'
import { Grid } from '@mui/material'
import { PageHeader } from 'app/components/PageHeader/PageHeader'

export const InvestCommitmentView = () => {
  const { commitmentId } = useParams<{ commitmentId: string }>()
  const { data, isLoading } = useCommitmentById(commitmentId)

  if (isLoading || data === undefined) {
    return null
  }

  return (
    <Grid container direction='column'>
      <Grid item>
        <PageHeader title={data.dso.tokenName} />
      </Grid>

      <Grid item>
        <RejectionMessage data={data} />
      </Grid>

      <Grid item>
        <CommitmentPreview data={data} isUserView />
      </Grid>
    </Grid>
  )
}
