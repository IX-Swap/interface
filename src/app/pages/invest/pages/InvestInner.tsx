import { useInvestListRouter } from 'app/pages/invest/routers/investLandingRouter'

export const InvestInner = () => {
  const { renderRoutes } = useInvestListRouter()

  return renderRoutes()
}
