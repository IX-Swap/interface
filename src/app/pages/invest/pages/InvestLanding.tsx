import React from 'react'
import { useAuth } from 'hooks/auth/useAuth'
import { Grid, Button, Typography } from '@material-ui/core'
import { useSetPageTitle } from 'app/hooks/useSetPageTitle'
import { PromotedDSOs } from 'app/components/DSO/components/PromotedDSOs'
import { AppRouterLinkComponent } from 'components/AppRouterLink'
import { useInvestRouter } from 'app/pages/invest/routers/router'
import { VSpacer } from 'components/VSpacer'
import { DSOTable } from 'app/pages/invest/components/DSOTable/DSOTable'
import { PromoBanner } from '../components/PromoBanner'

export const InvestLanding = () => {
  const { user } = useAuth()
  const { paths } = useInvestRouter()

  useSetPageTitle(`Welcome, ${user?.name ?? ''}`)

  return (
    <>
      <Grid container justify='space-between'>
        <Grid item>
          <Typography variant='h4'>Top Offers</Typography>
        </Grid>
        <Grid item>
          <Button
            component={AppRouterLinkComponent}
            to={paths.commitments}
            color='primary'
            variant='outlined'
            size='large'
            disableElevation
          >
            View my commitments
          </Button>
        </Grid>
      </Grid>
      <VSpacer size='small' />
      <PromotedDSOs />
      <VSpacer size='medium' />
      <PromoBanner />
      <VSpacer size='medium' />
      <Grid container direction='column' spacing={4}>
        <Grid item>
          <Typography variant='h4'>More Offers for You</Typography>
        </Grid>
        <Grid item>
          <DSOTable />
        </Grid>
      </Grid>
    </>
  )
}
