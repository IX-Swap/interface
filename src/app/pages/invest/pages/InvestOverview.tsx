import React from 'react'
import { Grid, Button, Typography, useTheme } from '@material-ui/core'
import { AppRouterLinkComponent } from 'components/AppRouterLink'
import { InvestRoute } from 'app/pages/invest/router/config'
import { VSpacer } from 'components/VSpacer'
import { PromotedDSOs } from 'app/components/DSO/components/PromotedDSOs'
import { PromoBanner } from 'app/pages/invest/components/PromoBanner'
import { SecondaryMarketTable } from 'app/pages/invest/components/SecondaryMarketTable/SecondaryMarketTable'

export const InvestOverview = () => {
  const theme = useTheme()

  return (
    <>
      <Grid container justify='space-between'>
        <Grid item container justify={'flex-end'}>
          <Button
            component={AppRouterLinkComponent}
            to={InvestRoute.commitments}
            color='primary'
            variant='outlined'
            size='large'
            disableElevation
            style={{ color: theme.palette.primary.main }}
          >
            My commitments
          </Button>
        </Grid>
        <PromoBanner />
        <VSpacer size='medium' />
      </Grid>
      <VSpacer size='small' />
      <PromotedDSOs />
      <VSpacer size='medium' />

      <Grid container direction='column' spacing={4}>
        <Grid item>
          <Typography variant='h4'>Secondary Market</Typography>
        </Grid>
        <Grid item>
          <SecondaryMarketTable />
        </Grid>
      </Grid>
    </>
  )
}
