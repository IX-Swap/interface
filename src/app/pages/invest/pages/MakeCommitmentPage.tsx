import { Box, Grid } from '@mui/material'
import { PageHeader } from 'app/components/PageHeader/PageHeader'
import { MakeCommitment } from 'app/pages/invest/components/MakeCommitment/MakeCommitment'
import React from 'react'

export const MakeCommitmentPage = () => {
  return (
    <Box width='100%'>
      <Grid container spacing={0}>
        <Grid item xs={12}>
          <PageHeader title='Make Commitment' />
        </Grid>
        <Grid item xs={12}>
          <MakeCommitment />
        </Grid>
      </Grid>
    </Box>
  )
}
