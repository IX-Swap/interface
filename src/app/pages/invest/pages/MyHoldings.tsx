import { Grid, Typography } from '@material-ui/core'
import { PageHeader } from 'app/components/PageHeader/PageHeader'
import React from 'react'

export const MyHoldings = () => {
  return (
    <Grid container direction='column' spacing={3}>
      <Grid item>
        <PageHeader title='My Holdings' showBreadcrumbs />
      </Grid>
      <Grid item>
        <Typography variant='h3'>Your Holdings</Typography>
        <Typography variant='body1'>
          View, manage and track the value of your private company shares and
          stock options over time. Receive insights, and investment and
          liquidity opportunities specific to your holdings.
        </Typography>
      </Grid>
      <Grid item></Grid>
    </Grid>
  )
}
