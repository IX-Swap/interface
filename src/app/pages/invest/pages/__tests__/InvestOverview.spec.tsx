import React from 'react'
import { render } from 'test-utils'
import { InvestOverview } from 'app/pages/invest/pages/InvestOverview'

describe('InvestOverview', () => {
  it('should match snapshot', () => {
    const { container } = render(<InvestOverview />)
    expect(container).toMatchSnapshot()
  })
})
