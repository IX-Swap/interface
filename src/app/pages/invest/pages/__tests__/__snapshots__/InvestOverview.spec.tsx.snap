// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InvestOverview should match snapshot 1`] = `
<div>
  <div
    class="makeStyles-wrapper-5 MuiBox-root css-0"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-127nfur-MuiContainer-root"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 css-1b3w61-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item css-4pkl32-MuiGrid-root"
        >
          <div
            class="MuiBox-root css-mkpo7v"
          >
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column css-1o4kjq1-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
              >
                <div
                  class="MuiBox-root css-geek62"
                >
                  <h2
                    class="MuiTypography-root MuiTypography-h2 makeStyles-title-6 css-rldqdm-MuiTypography-root"
                  >
                    Overview
                  </h2>
                </div>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
              >
                <div
                  class="MuiBox-root css-ftbd7"
                >
                  <nav
                    class="MuiTypography-root MuiTypography-body1 MuiBreadcrumbs-root css-128kl2r-MuiTypography-root-MuiBreadcrumbs-root"
                  >
                    <ol
                      class="MuiBreadcrumbs-ol css-4pdmu4-MuiBreadcrumbs-ol"
                    />
                  </nav>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
        >
          <div
            class="MuiBox-root css-uwwqev"
          >
            <div
              class="MuiGrid-root MuiGrid-container css-1k71wqj-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-container css-1c8dwhi-MuiGrid-root"
                >
                  <div
                    class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
                  >
                    <div
                      class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-fullWidth makeStyles-wrapper-9 css-111g48h-MuiInputBase-root-MuiOutlinedInput-root"
                    >
                      <div
                        aria-expanded="false"
                        aria-haspopup="listbox"
                        class="makeStyles-select-10 MuiSelect-select MuiSelect-outlined MuiOutlinedInput-input MuiInputBase-input css-1ub9as-MuiSelect-select-MuiInputBase-input-MuiOutlinedInput-input"
                        role="button"
                        tabindex="0"
                      >
                        Select Market
                      </div>
                      <input
                        aria-hidden="true"
                        class="MuiSelect-nativeInput css-yf8vq0-MuiSelect-nativeInput"
                        tabindex="-1"
                        value="Select Market"
                      />
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiSelect-icon MuiSelect-iconOutlined css-7glybp-MuiSvgIcon-root-MuiSelect-icon"
                        data-testid="ArrowDropDownIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M7 10l5 5 5-5z"
                        />
                      </svg>
                      <fieldset
                        aria-hidden="true"
                        class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                      >
                        <legend
                          class="css-nnbavb"
                        >
                          <span
                            class="notranslate"
                          >
                            ​
                          </span>
                        </legend>
                      </fieldset>
                    </div>
                  </div>
                  <div
                    class="MuiGrid-root MuiGrid-item css-6prgu6-MuiGrid-root"
                    data-testid="primarySearch"
                  >
                    <div
                      class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-adornedStart css-mhayhs-MuiInputBase-root-MuiOutlinedInput-root"
                    >
                      <div
                        class="MuiInputAdornment-root MuiInputAdornment-positionStart css-ittuaa-MuiInputAdornment-root"
                      >
                        <span
                          class="notranslate"
                        >
                          ​
                        </span>
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root MuiSvgIcon-colorDisabled MuiSvgIcon-fontSizeMedium css-fuvw14-MuiSvgIcon-root"
                          data-testid="SearchIcon"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
                          />
                        </svg>
                      </div>
                      <input
                        class="MuiOutlinedInput-input MuiInputBase-input MuiInputBase-inputAdornedStart css-mmb06o-MuiInputBase-input-MuiOutlinedInput-input"
                        placeholder="Search"
                        type="text"
                        value=""
                      />
                      <fieldset
                        aria-hidden="true"
                        class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                      >
                        <legend
                          class="css-nnbavb"
                        >
                          <span
                            class="notranslate"
                          >
                            ​
                          </span>
                        </legend>
                      </fieldset>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item css-6prgu6-MuiGrid-root"
              >
                <a
                  class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeLarge MuiButton-outlinedSizeLarge MuiButton-disableElevation MuiButtonBase-root  css-pkrwml-MuiTypography-root-MuiLink-root-MuiButtonBase-root-MuiButton-root"
                  href="/app/invest/commitments"
                  style="padding-right: 0px; padding-left: 0px;"
                  tabindex="0"
                >
                  My Commitments
                   
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-1yj5d43-MuiTypography-root"
                  >
                    0
                  </p>
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-p03apy-MuiContainer-root"
  >
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-6 MuiGrid-direction-xs-column css-167wck0-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-item css-11wtnxd-MuiGrid-root"
      />
      <div
        class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
      />
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-spacing-xs-4 MuiGrid-direction-xs-column css-1qixtcg-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
        >
          <h4
            class="MuiTypography-root MuiTypography-h4 css-vxrdkt-MuiTypography-root"
          >
            Secondary Market
          </h4>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
          style="max-width: 100%;"
        >
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 MuiGrid-direction-xs-column css-32q0xd-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-item css-9pwih8-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-5 MuiGrid-grid-md-4 MuiGrid-grid-lg-3 css-b7hlqe-MuiGrid-root"
              >
                <div
                  class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-adornedEnd css-zb37ev-MuiInputBase-root-MuiOutlinedInput-root"
                  data-testid="secondaryMarketSearch"
                >
                  <input
                    class="MuiOutlinedInput-input MuiInputBase-input MuiInputBase-inputAdornedEnd css-1vvkud8-MuiInputBase-input-MuiOutlinedInput-input"
                    placeholder="Search"
                    type="text"
                    value=""
                  />
                  <div
                    class="MuiInputAdornment-root MuiInputAdornment-positionEnd css-1laqsz7-MuiInputAdornment-root"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-colorDisabled MuiSvgIcon-fontSizeMedium css-fuvw14-MuiSvgIcon-root"
                      data-testid="SearchIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
                      />
                    </svg>
                  </div>
                  <fieldset
                    aria-hidden="true"
                    class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                  >
                    <legend
                      class="css-nnbavb"
                    >
                      <span
                        class="notranslate"
                      >
                        ​
                      </span>
                    </legend>
                  </fieldset>
                </div>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-auto css-1hsbqbm-MuiGrid-root"
              >
                <a
                  class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeLarge MuiButton-outlinedSizeLarge MuiButton-disableElevation MuiButtonBase-root  css-1dz9je6-MuiTypography-root-MuiLink-root-MuiButtonBase-root-MuiButton-root"
                  href="/app/accounts/holdings"
                  style="color: rgb(76, 136, 255);"
                  tabindex="0"
                >
                  My Holdings
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </a>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
              style="overflow-x: scroll; max-width: 100%;"
            >
              <div
                class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column css-1vpwcmr-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
                >
                  <span
                    class="MuiLinearProgress-root MuiLinearProgress-colorPrimary MuiLinearProgress-indeterminate css-6gra9t-MuiLinearProgress-root"
                    role="progressbar"
                  >
                    <span
                      class="MuiLinearProgress-bar MuiLinearProgress-barColorPrimary MuiLinearProgress-bar1Indeterminate css-1y2arm0-MuiLinearProgress-bar1"
                    />
                    <span
                      class="MuiLinearProgress-bar MuiLinearProgress-barColorPrimary MuiLinearProgress-bar2Indeterminate css-fng5ps-MuiLinearProgress-bar2"
                    />
                  </span>
                  <div
                    class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded css-1nt1r1b-MuiPaper-root"
                    style="background-color: inherit;"
                  >
                    <div
                      class="MuiTableContainer-root css-rorn0c-MuiTableContainer-root"
                    >
                      <table
                        aria-label="table"
                        class="MuiTable-root css-7mlt34-MuiTable-root"
                        data-testid="table"
                      >
                        <thead
                          class="MuiTableHead-root css-15wwp11-MuiTableHead-root"
                          style="background-color: rgb(20, 18, 114); height: 50px; display: table-header-group;"
                        >
                          <tr
                            class="MuiTableRow-root MuiTableRow-head css-8dfetm-MuiTableRow-root"
                          >
                            <th
                              class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1efwvfh-MuiTableCell-root"
                              scope="col"
                            >
                              <b
                                style="color: rgb(255, 255, 255);"
                              />
                            </th>
                            <th
                              class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1efwvfh-MuiTableCell-root"
                              scope="col"
                            >
                              <b
                                style="color: rgb(255, 255, 255);"
                              >
                                Pair
                              </b>
                            </th>
                            <th
                              class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1efwvfh-MuiTableCell-root"
                              scope="col"
                            >
                              <b
                                style="color: rgb(255, 255, 255);"
                              >
                                Symbol
                              </b>
                            </th>
                            <th
                              class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1efwvfh-MuiTableCell-root"
                              scope="col"
                            >
                              <b
                                style="color: rgb(255, 255, 255);"
                              >
                                Name
                              </b>
                            </th>
                            <th
                              class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-hm466y-MuiTableCell-root"
                              scope="col"
                            >
                              <b
                                style="color: rgb(255, 255, 255);"
                              >
                                Issued By
                              </b>
                            </th>
                            <th
                              class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignRight MuiTableCell-sizeMedium css-1v6mra9-MuiTableCell-root"
                              scope="col"
                            >
                              <b
                                style="color: rgb(255, 255, 255);"
                              >
                                Price
                              </b>
                            </th>
                            <th
                              class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1efwvfh-MuiTableCell-root"
                              scope="col"
                            >
                              <b
                                style="color: rgb(255, 255, 255);"
                              />
                            </th>
                            <th
                              class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1efwvfh-MuiTableCell-root"
                              scope="col"
                            >
                              <b
                                style="color: rgb(255, 255, 255);"
                              >
                                Type
                              </b>
                            </th>
                            <th
                              class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1efwvfh-MuiTableCell-root"
                              scope="col"
                            >
                              <b
                                style="color: rgb(255, 255, 255);"
                              />
                            </th>
                          </tr>
                        </thead>
                        <tbody
                          class="MuiTableBody-root css-apqrd9-MuiTableBody-root"
                        />
                      </table>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
